api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -421,5 +421,15 @@ public void purgeUserApp(UserApp userApp) {
         new AppFrameworkActivator().contextRefreshed();
     }
 
+	@Override
+	public List<Extension> getAllExtensions() {
```

Comment:
It would better to add a test for this new function in AppFrameworkServiceImplTest.java